src/SlackHookHandler.ts

```diff
@@ -107,6 +108,9 @@ export class SlackHookHandler extends BaseSlackHandler {
                 }
             } catch (e) {
                 log.error("SlackHookHandler failed:", e);
+                if (res.finished) {
```

Comment:
Above this line should be a comment saying "Do not send error if HTTP connection is closed".

If the issue was important, maybe link it.